cmd/cloudx/client/handler_test.go

```diff
@@ -75,6 +75,10 @@ func TestCommandHelper(t *testing.T) {
 			p, err := loggedIn.GetProject(project)
 			require.NoError(t, err)
 			assertValidProject(t, p)
+
+			p, err = loggedIn.GetProject(p.Slug)
+			require.NoError(t, err)
+			assertValidProject(t, p)
```

Comment:
Just a minor improvement to ensure we are getting the right project.
```suggestion
			actual, err := loggedIn.GetProject(p.Slug)
			require.NoError(t, err)
			assert.Equal(t, p, actual)
```
If this does not work, you could add an assertion on the project ID.